drive_to_gps/msg/heading.msg

```diff
@@ -0,0 +1,2 @@
+bool valid
```

Comment:
i don't think a `valid field` provides any value. if its not valid, it shouldn't be published?